xml/admin_monitoring_alerting.xml

```diff
@@ -1,1058 +1,374 @@
-   <para>
-    &alertmanager;'s configuration is different for each deployment. Therefore,
-    &deepsea; does not ship any related defaults. You need to provide your own
-    <filename>alertmanager.yml</filename> configuration file. The
-    <package>alertmanager</package> package by default installs a
-    configuration file <filename>/etc/prometheus/alertmanager.yml</filename>
-    which can serve as an example configuration. If you prefer to have your
-    &alertmanager; configuration managed by &deepsea;, add the following key to
-    your pillar, for example to the
-    <filename>/srv/pillar/ceph/stack/ceph/minions/<replaceable>YOUR_SALT_MASTER_MINION_ID</replaceable>.sls</filename>
-    file:
-   </para>
-<!--   <para>
-    For a complete example of &alertmanager;'s configuration file, see
-    <xref linkend="bp-troubleshooting-alerts"/>.
-   </para>-->
-<screen>
-monitoring:
- alertmanager_config:
-   /path/to/your/alertmanager/config.yml
-</screen>
-   <para>
-    &alertmanager;'s configuration file is written in the YAML format. It
-    follows the scheme described below. Parameters in brackets are optional.
-    For non-list parameters the default value is used. The following generic
-    placeholders are used in the scheme:
-   </para>
-   <variablelist>
-    <varlistentry>
-     <term><replaceable>DURATION</replaceable></term>
-     <listitem>
-      <para>
-       A duration matching the regular expression
-       <literal>[0-9]+(ms|[smhdwy])</literal>
-      </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term><replaceable>LABELNAME</replaceable></term>
-     <listitem>
-      <para>
-       A string matching the regular expression
-       <literal>[a-zA-Z_][a-zA-Z0-9_]*</literal>
-      </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term><replaceable>LABELVALUE</replaceable></term>
-     <listitem>
+  <para>
+    We recommend that the <systemitem class="daemon">Node exporter</systemitem>
+    will be deployed on all nodes. This can be done using the
+    <filename>monitoring.yaml</filename> file with the
+    <literal>node-exporter</literal> service type. See <xref
+    linkend="monitoring-stack-deploy" /> for more information on deploying
+    services.
+  </para>
+
+  <sect1 xml:id="monitoring-stack-deploy">
+    <title>Deploying Monitoring With &cephadm;</title>
+    <para>
+      To deploy the monitoring stack you need to either use a YAML configuration
+      file or the CLI. Whichever method you will use, you need to ensure the
+      &prometheusmgr; is enabled.
+    </para>
+    <para>
+      The &prometheusmgr; needs to be enabled using the following command:
+    </para>
+    <screen>&prompt.cephuser;ceph mgr module enable prometheus</screen>
+    <note>
       <para>
-       A string of Unicode characters.
+        Ensure this command is run before &prometheus; is deployed. If the
+        <command>ceph mgr module enable prometheus</command> was not run before
```

Comment:
2 alternative suggestions:

Either remove "the" before the command, which is definitely wrong, but still leaves it very wordy

**or**

Replace the command string with just "command" -- i.e.,

"if the command was not run before..."